deltacat/compute/compactor/compaction_session.py

```diff
@@ -403,11 +462,28 @@ def _execute_compaction_round(
         enable_profiler=enable_profiler,
         metrics_config=metrics_config,
     )
+
+    dedupe_invoke_end = time.monotonic()
     logger.info(f"Getting {len(dd_tasks_pending)} dedupe results...")
     dd_results: List[DedupeResult] = ray.get(dd_tasks_pending)
     logger.info(f"Got {len(dd_results)} dedupe results.")
+
+    dedupe_results_retrieved_at = time.time()
```

Comment:
Minor: Can we leave a comment here about why we use time.time() in this case vs. other timing mechanisms? I can imagine someone inadvertently breaking this in the future and it being easily missed during the code review...